packages/uploadthing/src/internal/handler.ts

```diff
@@ -309,10 +309,11 @@ export const buildRequestHandler = <
             (f) =>
               isObject(f) &&
               typeof f.name === "string" &&
-              typeof f.size === "number",
+              typeof f.size === "number" &&
+              typeof f.type === "string",
```

Comment:
idk if we should send this from client or use lookup using filename, but i think it should be included in the `opts.files` you get in `.middleware()`.